static/js/weather-components.js

```
@@ -910,13 +1592,13 @@ class WeatherApp {
         try {
             const response = await fetch(`https://nominatim.openstreetmap.org/reverse?format=json&lat=${lat}&lon=${lon}&zoom=10&addressdetails=1`);
             const data = await response.json();
-            
+
             if (data && data.address) {
                 const address = data.address;
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)